test/ethers.js/ERC20.js

```diff
@@ -68,7 +68,8 @@ describe("ERC20", function () {
 
     ownerBalance.should.be.equal(TOKEN_INITIAL_SUPPLY);
   });
-  it("Should be able to transfer token", async function () {
```

Comment:
I'm happy to do this temporarily so we can make progress, but we should have a running issue where we record these.